scipy/cluster/tests/test_hierarchy.py

```diff
@@ -1260,3 +1260,12 @@ def test_Heap(xp):
     pair = heap.get_min()
     assert_equal(pair['key'], 1)
     assert_equal(pair['value'], 10)
+
+
+@skip_xp_backends(np_only=True, reasons=['scipy specific corner case'])
```

Comment:
This function supports cpu backends so I think we should do this?
```suggestion
@skip_xp_backends(cpu_only=True)
```

Reply:
Thanks, applied in dc8d1b524fea8234ff6609ac748ec8317de4e9c1.